adoc/admin-security-certificates.adoc

```diff
@@ -243,6 +287,22 @@ helm install suse/cert-exporter \
     --set customSecret.certs[1].includeKeys="{ca.crt,tls.crt}" \
     --set customSecret.certs[1].labelSelector="{key=value}"
 ----
+Or if you have selected the Helm 3 alternative <<helm-tiller-install>>:
++
+[source,bash]
+----
+helm install suse/cert-exporter ${RELEASE_NAME} \
```

Comment:
In Helm3, the name should come before chart.

Reply:
done